src/api/types/rest.ts

```diff
@@ -473,7 +473,7 @@ export interface Collection extends CollectionBase {
     data: number[];
 }
 
-export function isCollection(object: Collection | SampleBase | Data): object is Collection {
+export function isCollection(object: CollectionBase | SampleBase | Data): object is CollectionBase {
     return object.hasOwnProperty('description') &&
         object.hasOwnProperty('settings') &&
         object.hasOwnProperty('data') &&
```

Comment:
Fixed and moved to a separate pullrequest: https://github.com/genialis/resolwe-js/pull/35
We will need to merge that pullrequest much quicker than this one